scaffolds.json

```diff
@@ -16,5 +16,19 @@
     "repo": "https://github.com/onflow/scaffold-flow.git",
     "description": "Simple demo application using next.js and FCL with provided Cadence contracts.",
     "commit": "4158c5e11b6d2c9e1cbc99af597e3b42b2e6c193"
+  },
+  {
+    "name": "Simple Unity",
+    "repo": "https://github.com/onflow/UnityFlowSDK.git",
+    "description": "Simple examples how to interact with Flow network using Unity SDK.",
+    "commit": "d01a4b14a98383c18b4d57e87624cb8dba6d0946",
+    "folder": "Samples~/FlowControlQuickstartCodeSample"
+  },
+  {
+    "name": "Mobile Unity Game",
+    "repo": "https://github.com/onflow/UnityFlowSDK.git",
+    "description": "Example Unity words game built on Flow using the Unity SDK.",
```

Comment:
Don't need to say Unity twice, could just be: `"Example word game built on Flow using the Unity SDK."`

Reply:
thank you! this is better for sure

Reply:
addressed